addons/project/views/project_sharing_project_task_views.xml

```diff
@@ -294,6 +294,11 @@
         <field name="view_mode">kanban,tree,form</field>
         <field name="search_view_id" ref="project.project_sharing_project_task_view_search"/>
         <field name="domain">[('project_id', '=', active_id), ('display_in_project', '=', True)]</field>
+        <field name="context">{
+            'default_project_id': active_id,
+            'active_id_chatter': active_id,
+            'delete': false,
+        }</field>
```

Comment:
Not something that I think should change (just curious) but why is this needed here ? :thinking:

Reply:
Before my changes, the additional context was given in the python method (see [here](https://github.com/odoo/odoo/blob/b2c33e01f3fdd7ecda891b804af8cc9b4572933f/addons/project/models/project_project.py#L641-L646)), but since I remove that method, I have to pass the context in that action to keep the context we got before. 🙂 

(Also, I see I forgot to add `'search_default_open_tasks': True,` in the context, I will addd it)